### PR TITLE
SPCTR-3094 : Implemented Global block style in core blocks

### PR DESCRIPTION
**Pull request by rahulvarma722**
### Description
Global Block Style is a robust and flexible styling solution designed to provide a cohesive look and feel across all blocks.
In this pr we have added global block style in  core blocks. 

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: blocks-config/uagb-controls/AddGBSStyles.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+				);
+			}
+		}, [globalBlockStyles] );
+
+		// Filter the placeholder attribute.
+
+		const modifiedAttr = { ...attributes };
+
+		for ( const objectKey in modifiedAttr ) {
+			if( 0.001020304 === modifiedAttr?.[objectKey] ){
```
Comment (aapatre) [inline blocks-config/uagb-controls/AddGBSStyles.js:47]: Please explain why the random number is used by adding a code-comment

Comment (aapatre) [inline blocks-config/uagb-controls/AddGBSStyles.js:47]: Also, we can declare this constant somewhere since it's being used throughout the code

Comment (rahulvarma722) [reply blocks-config/uagb-controls/AddGBSStyles.js]: Done

**Review by aapatre — APPROVED** [review]

File: classes/class-uagb-helper.php [hunk 2]
```diff
@@ -886,14 +886,53 @@ public static function get_typography_css( $attr, $slug, $selector, $combined_se
...
 		/**
 		 * Parse CSS into correct CSS syntax.
 		 *
 		 * @param array  $combined_selectors The combined selector array.
 		 * @param string $id The selector ID.
+		 * @param array  $gbs_attributes The GBS attributes array.
+		 * 
+		 * @return array Responsive CSS.
 		 * @since 1.15.0
 		 */
```
Comment (AbhijitNage123) [inline classes/class-uagb-helper.php:930]: @rahulvarma722 Could you please check PHPDoc comment as per this format - https://docs.google.com/document/d/1pcSs5OrvHrJG5DDKPeInU5H_JGZz_sa2ylD1LGSVoB0/edit#heading=h.yvfss3t6ia8p. Please maintain consistency everywhere

**Review by AbhijitNage123 — APPROVED** [review]

**Pull request merged**